apps/catalogue3/src/views/DatasetMappingsView.vue

```diff
@@ -49,24 +49,22 @@
               ? datasetmappings.description
               : "N/A"
           }}
+          <RowButtonEdit
```

Comment:
this leads to the ux-idea of looking at data in 'view' mode and having the option to edit a field ( i experimented with this in the security manager in emx1, personally i thought the UX was nice (copied the idea for the Sentry UI management UI)), using components it's not more expensive to implement ( if you already have the row-Add/wizard option for adding new rows),

Reply:
Indeed, in this case it is only one field and might be done differently.